voucher/src/main/java/com/adyen/checkout/voucher/internal/ui/view/FullVoucherView.kt

```diff
@@ -67,47 +77,37 @@ class FullVoucherView @JvmOverloads constructor(
         initLocalizedStrings(localizedContext)
 
         observeDelegate(delegate, coroutineScope)
+        this.coroutineScope = coroutineScope
 
         binding.buttonCopyCode.setOnClickListener { copyCode(delegate.outputData.reference) }
-        binding.buttonDownloadPdf.setOnClickListener { delegate.downloadVoucher(context) }
     }
 
     private fun initLocalizedStrings(localizedContext: Context) {
-        binding.textViewIntroduction.setLocalizedTextFromStyle(
-            R.style.AdyenCheckout_Voucher_Description_Boleto,
-            localizedContext
-        )
         binding.textViewPaymentReference.setLocalizedTextFromStyle(
             R.style.AdyenCheckout_Voucher_PaymentReference,
-            localizedContext
-        )
-        binding.buttonCopyCode.setLocalizedTextFromStyle(
-            R.style.AdyenCheckout_Voucher_ButtonCopyCode,
-            localizedContext
-        )
-        binding.buttonDownloadPdf.setLocalizedTextFromStyle(
-            R.style.AdyenCheckout_Voucher_ButtonDownloadPdf,
-            localizedContext
-        )
-        binding.textViewExpirationLabel.setLocalizedTextFromStyle(
-            R.style.AdyenCheckout_Voucher_ExpirationDateLabel,
-            localizedContext
+            localizedContext,
```

Comment:
Localization with standalone components does not fully work with the new labels.
The best way to test this is to set the locale in the test app settings to a different value than the device locale, then use the instant payment component to test multibanco, you should see some strings are still in English.

You can check the `initLocalizedStrings` method and the `localizedContext` variable to see how we support this currently (you can find them in every view) but I will explain more in person.